lib/ansible/playbook/task.py

```diff
@@ -77,6 +77,7 @@ class Task(Base, Conditional, Taggable, CollectionSearch):
     _delegate_to = FieldAttribute(isa='string')
     _delegate_facts = FieldAttribute(isa='bool')
     _failed_when = FieldAttribute(isa='list', default=list)
+    _implicit = FieldAttribute(isa='bool', default=False, private=True, inherit=False)
```

Comment:
This would allow the field `implicit` to be set on a task rather than being an error. It should be internal, so it should probably be a task instance attribute instead, similar to my PR to add tag support for explicit meta tasks.

Reply:
Yep, I agree, it should just be an instance attr, and not a FA. To my knowledge, `private` isn't actually used anywhere, so it doesn't prevent someone from setting this on a task in a playbook.

Reply:
it is used in keyword doc generation .. but we should add it also to the ds creation when reading the pb source

Reply:
@s-hertel that was my original thought, but talked with @bcoca about it and we came up with the above.
I do think an attribute is probably the better route here and will update it.

Reply:
Though I also agree with @bcoca that `private` should do what it says. :slightly_smiling_face: